imsim/bandpass.py

```diff
@@ -48,9 +178,22 @@ def buildBandpass(self, config, base, logger):
             the constructed Bandpass object.
         """
         req = { 'band' : str }
-        kwargs, safe = galsim.config.GetAllParams(config, base, req=req)
+        opt = {
+            'airmass' : float,
+            'camera' : str,
+            'det_name' : str
+        }
+        kwargs, safe = galsim.config.GetAllParams(
+            config, base, req=req, opt=opt
+        )
         kwargs['logger'] = logger
         bp = RubinBandpass(**kwargs)
+
+        # Also, store the kwargs=None version in the base config.
+        if 'airmass' in kwargs:
```

Comment:
Should this also check for `camera` in `kwargs`?

Reply:
Yep.  But it's even easier and more future proof to just skip the check entirely always, computing the fiducial_bandpass (can't imagine this is actually a time sink).